website/static/css/addonsettings.css

```diff
@@ -16,3 +16,11 @@
 .filebrowser #tb-tbody {
     height: 300px;
 }
+
+p.current-folder{
+    word-wrap: break-word;
+}
+
+h4#connect-folder{
```

Comment:
This is a brittle selector (what if we change h4 to h3)? Also, it might make sense to make `connect-folder` a class rather than an ID, because it will appear more than once on the page.